.github/settings.yml

```diff
@@ -0,0 +1,68 @@
+---
+repository:
+  name: openshift
+  description: OpenShift catalog templates for ownCloud
+  topics: docker, openshift
+
+  private: false
+  has_issues: true
+  has_projects: false
+  has_wiki: false
+  has_downloads: false
+
+  default_branch: master
+
+  allow_squash_merge: true
+  allow_merge_commit: true
+  allow_rebase_merge: true
+
+labels:
+  - name: bug
+    color: d73a4a
+    description: Something isn't working
+  - name: documentation
+    color: 0075ca
+    description: Improvements or additions to documentation
+  - name: duplicate
+    color: cfd3d7
+    description: This issue or pull request already exists
+  - name: enhancement
+    color: a2eeef
+    description: New feature or request
+  - name: good first issue
+    color: 7057ff
+    description: Good for newcomers
+  - name: help wanted
+    color: 008672
+    description: Extra attention is needed
+  - name: invalid
+    color: e4e669
+    description: This doesn't seem right
+  - name: question
+    color: d876e3
+    description: Further information is requested
+  - name: wontfix
+    color: ffffff
+    description: This will not be worked on
+
+teams:
+  - name: bot
+    permission: admin
+
+branches:
+  - name: master
+    protection:
+      required_pull_request_reviews:
+        required_approving_review_count: 1
```

Comment:
this means you want to go with renovate approve bot?

Reply:
Not sure how to do handle it in owncloud-docker orga. While owncloud-ci isn't that critical owncloud-docker might be and we maybe want to enforce review in general. What do you think?

Reply:
oh I didn't notice that we ware in the owncloud-docker orga. But yes we should enforce reviews even for renovate in this case

Reply:
:+1: I'll enable renovate in a second PR